pkg/services/folder/folderimpl/folder_test.go

```diff
@@ -334,7 +353,10 @@ func TestFolderService(t *testing.T) {
 	})
 }
 
-func TestCreate_NestedFolders(t *testing.T) {
+func TestIntegrationNestedFolderService(t *testing.T) {
+	if testing.Short() {
```

Comment:
I'm curious about this, because I've seen it before and was surprised. These aren't integration tests as I understand them.  The tests use the mock stores - there's no integration. Am I missing something about how we use this flag?

Reply:
Correct, this should not be an integration test and it should not use `testing.Short` check here. Many other tests that use `InitTestDB` etc should use it though.